Update docs (#100)
dls-controls/tickit: docs/user/explanations/devices.rst

Devices
=======

Tickit simulations revolve around devices. Devices are user implemented classes
which behaviours mimic the hardware you wish to simulate.

> Any new device created must be of type device, have an update method, and must

Reviewer: ```suggestion
Any new device created must extend ``Device``, have an update method, and must
```
